src/systemtask/SystemMonitor.cpp

```diff
@@ -1,9 +1,9 @@
 #include "systemtask/SystemTask.h"
 #if configUSE_TRACE_FACILITY == 1
-// FreeRtosMonitor
-#include <FreeRTOS.h>
-#include <task.h>
-#include <nrf_log.h>
+  // FreeRtosMonitor
+  #include <FreeRTOS.h>
+  #include <task.h>
+  #include <nrf_log.h>
```

Comment:
This doesn't seem right... Should the preprocessor directives really be indented?

Reply:
IndentPPDirectives is set to BeforeHash in .clang-format, so it is working as intended at least. Do you think it should be changed?

Reply:
Just tried it out with IndentPPDirectives set to None, and it made many preprocessor if statements worse, so I'll leave it as is.